lib/api/controllers/routerController.js

```diff
@@ -418,15 +215,44 @@ module.exports = function RouterController (kuzzle) {
     api.get('/', (request, response) => {
       var _routes = {};
 
-      extractRouteInfo(_routes, routes);
-      extractRouteInfo(_routes, kuzzle.pluginsManager.routes, '/_plugin');
+      extractRouteInfo(_routes, routes, '/' + kuzzle.config.apiVersion);
+      extractRouteInfo(_routes, [{verb: 'get', url: '/_serverInfo', controller: 'read', action: 'serverInfo'}]);
+      extractRouteInfo(_routes, kuzzle.pluginsManager.routes, '/' + kuzzle.config.apiVersion + '/_plugin');
 
       response.setHeader('Access-Control-Allow-Origin', '*');
       response.setHeader('Access-Control-Allow-Headers', 'X-Requested-With');
       response.writeHead(200, {'Content-Type': 'application/json'});
       response.end(stringify({status: 200, error: null, result: {message: 'Available routes for this API version by verb.', routes: _routes}}));
     });
 
+    apiBase.get('/swagger.json', (request, response) => {
+      var content;
+      try {
+        content = fs.readFileSync(__dirname + '/../../../kuzzle-swagger.json');
```

Comment:
Exposing a `sync` FS operation to a HTTP API route is literally asking for problems: anyone executing this route will **freeze** NodeJS' callstack and Kuzzle during the time the file is read.

This PR can't go live until this has been replaced with an async fie read.